src/chat-bubble/interfaces.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export interface ChatBubbleProps {
+  /** Avatar slot paired with the chat bubble. */
```

Comment:
Do we call "chat bubble" a message area only or the entire component?

Reply:
Here I mean the message area but if I say message area instead, not sure if it would be clear?

Reply:
Maybe "Avatar slot paired with the chat bubble content"?